include/pisa/term_map.hpp

```diff
@@ -17,22 +20,26 @@ class TermMap {
     TermMap& operator=(TermMap&&);
     virtual ~TermMap();
 
-    [[nodiscard]] virtual auto operator()(std::string_view term) -> std::optional<std::uint32_t> = 0;
-    [[nodiscard]] virtual auto operator()(std::string const& term)
+    [[nodiscard]] virtual auto find(std::string_view term) const -> std::optional<std::uint32_t> = 0;
```

Comment:
We should note this API change explicitly for anyone using the lib.

Reply:
Added a `Changelog-changed:` trailer to the git message.